build-logic/module-identity/src/main/kotlin/gradlebuild.module-identity.gradle.kts

```diff
@@ -121,21 +116,6 @@ fun Project.buildTimestamp(): Provider<String> =
     }
 
 
-fun Project.buildTimestampFromBuildReceipt(): Provider<String> =
-    providers.of(BuildTimestampFromBuildReceiptValueSource::class) {
-        parameters {
-            ignoreIncomingBuildReceipt.set(project.ignoreIncomingBuildReceipt)
-            buildReceiptFileContents.set(
-                repoRoot()
-                    .dir("incoming-distributions")
```

Comment:
Now that we no longer consume the build timestamp from prior builds, I don't think we need to continue copying the build receipt to this dir during our builds. See:
* `upload-artifact` and `download-artifact` in the [GitHub workflows](https://github.com/gradle/gradle/blob/0c11c8be7e8e28b7920559bee0e3dc73740f12e0/.github/workflows/contributor-pr.yml#L80-L83)
* [Teamcity Build](https://github.com/gradle/gradle/blob/9dd917ab27e463cb56f01622df58058105855789/.teamcity/src/main/kotlin/common/extensions.kt#L229-L233)